src/Orleans.Core/Configuration/Options/StorageOptions.cs

```diff
@@ -0,0 +1,13 @@
+
+namespace Orleans.Hosting
+{
+    public class StorageOptions
+    {
+        public string DataConnectionString { get; set; }
+
+        /// <summary>
+        /// Set this property to override <see cref="DataConnectionString"/> for reminders.
+        /// </summary>
+        public string DataConnectionStringForReminders { get; set; }
```

Comment:
isn't think should belong to reminder option, instead of storage option? I'm so confused here.

Reply:
I don't know the history, or why this is the way it is, but it looks like there was a default account for storage, then an override was added for reminder storage, and each provider used it's own overrides for whichever default is used.  I didn't make this pattern, just ported it :/

Please keep in mind that refactoring these patterns are non-trivial, as there are expectations here used by various extensions, and we need to be able to map from the cluster configuration onto whatever new pattern we come up with.